### PR TITLE
[tf][py] Update function names

### PR DESCRIPTION
**Pull request by jpienaar**
Not sure how these are working, this should be without suffix (unless I messed up bumping).

### CODE REVIEW

**Comment by ShuaiShao93** [comment]
Can we add a unit test that can catch this? Preferably without using notebooks, so that it can catch the problems better

**Comment by pzread** [comment]
It looks like the job `python_release_packages` is hanging. I checked the runner and it seems idle. Some commands the runner is waiting on:

```
runner     11855   11842  0 15:20 ?        00:00:02 docker run --rm -v /runner-root/actions-runner/_work/iree/iree:/runner-root/actions-runner/_work/iree/iree -v /runner-root/actions-runner/_work/iree/iree/bindist:/runner-root/actions-runner/_work/iree/iree/bindist -e __MANYLINUX_BUILD_WHEELS_IN_DOCKER=1 -e override_python_versions=cp38-cp38 -e packages=iree-compiler -e package_suffix= -e output_dir=/runner-root/actions-runner/_work/iree/iree/bindist gcr.io/iree-oss/manylinux2014_x86_64-release@sha256:e83893d35be4ce3558c989e9d5ccc4ff88d058bc3e74a83181059cc76e2cf1f8 -- /runner-root/actions-runner/_work/iree/iree/build_tools/python_deploy/build_linux_packages.sh
root       34813   15100  0 15:31 ?        00:00:00 /bin/sh -c cd /runner-root/actions-runner/_work/iree/iree/compiler/build/cmake_build/runtime/src/iree/modules/check/test && /runner-root/actions-runner/_work/iree/iree/compiler/build/cmake_build/tools/iree-compile --output-format=vm-bytecode --mlir-print-op-on-diagnostic=false --iree-hal-target-backends=vmvx /runner-root/actions-runner/_work/iree/iree/runtime/src/iree/modules/check/test/success.mlir -o check_vmvx_local-task_success.mlir_module.vmfb --iree-hal-executable-object-search-path=\"/runner-root/actions-runner/_work/iree/iree/compiler/build/cmake_build\"
root       34829   34813  0 15:31 ?        00:00:00 /runner-root/actions-runner/_work/iree/iree/compiler/build/cmake_build/tools/iree-compile --output-format=vm-bytecode --mlir-print-op-on-diagnostic=false --iree-hal-target-backends=vmvx /runner-root/actions-runner/_work/iree/iree/runtime/src/iree/modules/check/test/success.mlir -o check_vmvx_local-task_success.mlir_module.vmfb --iree-hal-executable-object-search-path="/runner-root/actions-runner/_work/iree/iree/compiler/build/cmake_build"
```

**Comment by jpienaar** [comment]
Should I just rebase and upload to unstick?

**Comment by pzread** [comment]
> Should I just rebase and upload to unstick?

Yes, I think we can try

File: compiler/bindings/python/iree/compiler/__init__.py [hunk 1]
```diff
@@ -7,3 +7,4 @@
 # Re-export some legacy APIs from the tools package to this top-level.
 # TODO: Deprecate and remove these names once clients are migrated.
 from .tools import *
+version = 1
```
Comment (pzread) [inline compiler/bindings/python/iree/compiler/__init__.py]: What is the reason we need to put this?

Comment (jpienaar) [reply compiler/bindings/python/iree/compiler/__init__.py]: Oh I think this is is a bad snaphot, not intentional

**Review by pzread — APPROVED** [review]

**Pull request merged**